### ISSUE TITLE
[NBug] Value must not be null.

### ISSUE THREAD

**Issue by MarekKnapek** [gitextensions/gitextensions#10461]
### Environment

- Git Extensions 4.0.0.15569
- Build cba315df5366fca01240d93b6e5a5077fd5e6630
- Git 2.38.1.windows.1
- Microsoft Windows NT 10.0.20348.0
- .NET 6.0.10
- DPI 96dpi (no scaling)

### Issue description

```
System.Exception: Value must not be null.
   at Microsoft.Validates.Fail(String message)
   at GitUI.CommandsDialogs.SettingsDialog.SettingsTreeViewUserControl.textBoxFind_KeyUp(Object sender, KeyEventArgs e)
   at System.Windows.Forms.Control.OnKeyUp(KeyEventArgs e)
   at System.Windows.Forms.TextBox.OnKeyUp(KeyEventArgs e)
   at System.Windows.Forms.Control.ProcessKeyEventArgs(Message& m)
   at System.Windows.Forms.Control.ProcessKeyMessage(Message& m)
   at System.Windows.Forms.Control.WndProc(Message& m)
   at System.Windows.Forms.TextBox.WndProc(Message& m)
   at System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, WM msg, IntPtr wparam, IntPtr lparam)
```

Steps to reproduce:
 - Uninstall GitExtensions.
 - Do not restart computer.
 - Download and instal latest GitExtensions.
 - Do not restart computer.
 - Open GitExtensions.
 - Open some repo.
 - From menu bar, select Tools -> Options.
 - Value must not be null dialogue appears.

Marek.

### Steps to reproduce

Steps to reproduce:
 - Uninstall GitExtensions.
 - Do not restart computer.
 - Download and instal latest GitExtensions.
 - Do not restart computer.
 - Open GitExtensions.
 - Open some repo.
 - From menu bar, select Tools -> Options.
 - Value must not be null dialogue appears.

Marek.

**Comment by gerhardol** [comment]
I cannot recreate.
`textBoxFind_KeyUp` has no Validates and the code handles null what I see.

**Comment by RussKie** [comment]
This was fixed recently, and the fix will be in 4.0.1

**Comment by RussKie** [comment]
Duplicate of #10377
